openmrs/apps/clinical/app.json

```diff
@@ -43,6 +43,9 @@
           "programAttributes": [
           ],
           "personAttributes": [
+            "givenNameLocal",
```

Comment:
It can be repositioned next to name if we change stuff in bahmniapps, I don't mind it being up top to save screen space.